platform/device-manager.ts

```diff
@@ -0,0 +1,22 @@
+/**
+ * DeviceManager Stub
+ * TODO(cjdaly) integrate with Mock Local Home Platform and Mock Network
+ **/
+/// <reference types="@google/local-home-sdk" />
+
+export const DeviceManagerStub = class {
+  deviceId: string;
+  error: boolean;
+  public commands = new Array<smarthome.DataFlow.TcpRequestData>();
```

Comment:
does this need to be public? Maybe this could provide an assert to check if a given command was sent?

Reply:
removed `public`.  Also changed type to `smarthome.DataFlow.TcpRequestData[]` for consistency.  I'm planning on fleshing out this stub in a future PR (#23)